ovn-org/ovn-kubernetes
# WIP: Output address resolution requests for LoadBalancer IPs to LOCAL port

Currently address resolution requests (ARP/Neighbor solicitation) for a LoadBalancer IP
are answered by all of the gateway routers in the cluster.
By forwarding these requests to the local port, a network load balancer implementation
like MetalLB is able to be the only one replying to them - thus enabling it
to be the only one announcing a specific LoadBalancer service IP.

Signed-off-by: Ori Braunshtein <obraunsh@redhat.com>



**- What this PR does and why is it needed**


**- Special notes for reviewers**



**- How to verify it**
Using MetalLB for assigning LoadBalancer services an IP:
1. Deploy a DaemonSet/Deployment
2. Create a LoadBalancer Service for the set of pods, it gets assigned an IP by MetalLB.
3. arping/ndisc6 that IP from an interface that can reach the cluster via L2

The expected result is that only one reply is received.



**- Description for the changelog**
